library/unistd/spawnvpe.c

```diff
@@ -140,6 +140,7 @@ get_arg_string_length(char *const argv[]) {
 //     DebugPrintF("[child :] Done.");
 //     FreeSignal(ed->childSignal);
 // }
+#define USE_CNPT 0
```

Comment:
If we are going to use this new definition, we probably need to document it at the README file. Can you please add some info about what this is, what does it enable and how to enable it if needed? It would be great if that was part of this PR.

Reply:
Maybe this is overkill. I am primarily keeping the version of the text using CreateNewProcTags() for research reasons. At present only SystemTags will enable full functioning in cmake. CNPT gives the strange side effect, that the child will have its parents name in argv[0] (at least when the child uses newlib, as in the case of (g)make).

I could be convinced to ditch the CNPT code, or just replace it with #if 0.